.github/workflows/pre-release.yml

```diff
@@ -43,10 +64,6 @@ jobs:
       - name: Package Binaries
         run: yarn package
 
-      - name: Extract Informations
-        id: info
-        uses: botpress/gh-actions/extract_info@master
-
       - name: Rename Binary Files
         uses: botpress/gh-actions/rename_binaries@master
```

Comment:
I also forgot that `botpress/gh-actions/rename_binaries` renames binary with incorrect name...